cmd/file_namespace.go

```diff
@@ -144,7 +177,11 @@ routes:
 	namespaceCmd.Flags().StringVarP(&cmdNamespacePathPrefix, "path-prefix", "p", "",
 		"The path based namespace to apply.")
 	namespaceCmd.Flags().BoolVarP(&cmdNamespaceAllowEmptySelectors, "allow-empty-selectors",
-		"", false, "do not error out if the selectors return empty")
+		"", false, "Do not error out if the selectors return empty")
+	namespaceCmd.Flags().StringArrayVarP(&cmdNamespaceHosts, "host", "", []string{},
+		"Hostname to add for host-based namespacing. Repeat for multiple hosts.")
+	namespaceCmd.Flags().BoolVarP(&cmdClearHosts, "clear-hosts", "c", false,
+		"Clear existing hosts.")
```

Comment:
`--host=konghq.com` will append `konghq.com` to the existing `hosts` array of a route. `--clear-hosts` will clear the existing hosts. So combining the two flags has the net effect of replacing the `hosts` array, instead of appending the new one.

The flag description is "Clear existing hosts". Can you please provide an improved description for the flag, as you'd like to see it.?

Reply:
Why not provide a similar, but abbreviated, description you gave me in the above comment into the help text for users?